tools/c7n_oci/c7n_oci/actions/base.py

```diff
@@ -0,0 +1,253 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+import time
+from abc import ABC
+import abc
+
+from c7n.utils import type_schema
+from c7n.actions import BaseAction
+from c7n_oci.actions.utils import all_operation_completed
+
+log = logging.getLogger("custodian.oci.actions.base")
+
+
+class OCIBaseAction(BaseAction, ABC):
+    # OCIBaseAction handles the basic implementation of the OCI operation features
```

Comment:
Other classes use typical Python docstring conventions, but this one uses line comments instead. Is that intentional or carrying some semantic meaning?

Reply:
I will delete the commented part out.